docs/tooling_and_ecosystem/generators/matestack_page_generator.md

```diff
@@ -0,0 +1,52 @@
+# Matestack Page Generator
+
+## Usage
+
+Generates matestack pages to `app/matestack/pages`.
+
+## Parameters
+
+**NAME** - Mandatory. Creates an empty matestack component in `app/matestack/components/NAME.rb`.
```

Comment:
this seems wrong --> page instead of component, right?

Reply:
True that